thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyFunctionResource.java

```diff
@@ -19,6 +19,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@Path("thirdeye")
```

Comment:
anomaly-function is already a path in AnomalyResource, i.e. "/dashboard/anomaly-function" path is already present. I am planning to refactor function specific end points without touching the existing flow. Thats the reason why I chose /thirdeye/function for this end point, we can always rename this to a suitable one.